src/screens/LoginScreen/index.tsx

```diff
@@ -3,18 +3,28 @@ import { View } from 'react-native';
 import { style } from './style';
 import { SignInButton } from '../../components/SignInButton';
 import { useDispatch } from 'react-redux';
-import { signInSagaAction } from '../../redux/actions/authSagaActions/signInSagaAction';
+import { googleSignInSagaAction } from '../../redux/actions/authSagaActions/googleSignInSagaAction';
+import { fbSignInSagaAction } from '../../redux/actions/authSagaActions/fbSignInSagaAction';
 
 export const LoginScreen: FC = () => {
   const dispatch = useDispatch();
 
-  const googleSignIn = () => {
-    dispatch(signInSagaAction());
+  const googleObject = {
+    signInMethod: () => dispatch(googleSignInSagaAction()),
+    title: 'Sign in with Google',
+    typeStyle: 'google',
```

Comment:
if you provide type, make sense made all parameters configurable by type 

also typy should be enum